flow_screen_components/recordDetailFSC/force-app/main/default/lwc/recordDetailFlowScreenComponent/recordDetailFlowScreenComponent.html

```diff
@@ -1,8 +1,30 @@
+<!-- 
+Lightning Web Component for Flow Screens:       recordDetailFlowScreenComponent
 
+The RecordDetail component provides new ways to display record fields in Flow screens. 
```

Comment:
Let's keep all the docs in one place. Currently that's https://unofficialsf.com/the-recorddetail-component/

I like keeping the docs on the site and not here because the code can be intimidating. 

The argument against putting it in both places is that it falls out of sync and there's more to update...

Reply:
if you're worried about prematurely publishing changes, consider create a Page (not a Post) up on the site. that way the new content can be reviewed privately.